src/plots/cartesian/graph_interact.js

```diff
@@ -2099,8 +2099,7 @@ fx.dragElement = function(options) {
         if(options.doneFn) options.doneFn(gd._dragged, numClicks);
 
         if(!gd._dragged) {
-            var e2 = document.createEvent('MouseEvents');
-            e2.initEvent('click', true, true);
+            var e2 = new MouseEvent('click', { 'button': e.button });
```

Comment:
I don't believe event constructors like this (`new MouseEvent(..)`) [aren't fully supported in all browsers yet](https://msdn.microsoft.com/library/dn905219(v=vs.85).aspx), so we should stay with the "old" style.